Rakefile

```diff
@@ -1,20 +1,19 @@
 require 'rspec/core/rake_task'
 require 'rubygems/package_task'
+require 'mixlib/shellout/version'
 
-Dir[File.expand_path("../*gemspec", __FILE__)].reverse.each do |gemspec_path|
-  gemspec = eval(IO.read(gemspec_path))
+Dir[File.expand_path("../*gemspec", __FILE__)].each do |gemspec_path|
+  gemspec = eval(File.read(gemspec_path))
   Gem::PackageTask.new(gemspec).define
 end
 
-require 'mixlib/shellout/version'
-
 desc "Run all specs in spec directory"
 RSpec::Core::RakeTask.new(:spec) do |t|
   t.pattern = FileList['spec/**/*_spec.rb']
 end
 
 desc "Build it and ship it"
-task :ship => [:clean, :gem] do
+task ship: [:clobber_package, :gem] do
```

Comment:
I had gone down that route before.  rake/clean didn't get its clean targets from the Gem tasks.  Hence, running rake clean did nothing.  To get that to work, I had to manually add 'pkg' to CLEANS or just define my own clean => :clobber_package task.